src/com/itacademy/eshop/simulations/ManagerSimulation.java

```diff
@@ -20,32 +22,42 @@ public void simulate() {
         addThreeNewProducts();
         findAndRemoveOneProduct();
         changePriceForOneProduct();
-        removeAllProductsWithCategory(Category.FOOD);
+        removeAllProductsWithCategory(FOOD);
```

Comment:
you already have Category imported, you dont need to import FOOD separately, or, if you do, then you should do the same for lines 32-34 to be consistent